strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/validation/groupId/GroupIdValidationException.java

```diff
@@ -0,0 +1,27 @@
+package org.carlspring.strongbox.storage.validation.groupId;
+
+/**
+ * Created by dinesh on 12/6/17.
+ */
+public class GroupIdValidationException extends Exception {
```

Comment:
I think that a better name would be `LowercaseValidationException`, because there's also another task for validating the `artifactId` of the Maven artifact which will surely also throw an exception, so I think this exception class could be shared.